Index: src/includes/common/functions.php
===================================================================
--- src/includes/common/functions.php	(revision 6132)
+++ src/includes/common/functions.php	(working copy)
@@ -512,6 +512,9 @@

 		if ( current_user_can( 'read_private_topics' ) || current_user_can( 'edit_others_topics' ) || current_user_can( 'view_trash' ) ) {

+			// Setup the topics and topic_titles array
+			$topics = $topic_titles = array();
+
 			// Pending
 			$topics['pending'] = ( ! empty( $r['count_pending_topics'] ) && current_user_can( 'edit_others_topics' ) ) ? (int) $all_topics->{$pending} : 0;

@@ -548,6 +551,9 @@

 		if ( current_user_can( 'read_private_replies' ) || current_user_can( 'edit_others_replies' ) || current_user_can( 'view_trash' ) ) {

+			// Setup the replies and reply_titles array
+			$replies = $reply_titles = array();
+
 			// Pending
 			$replies['pending'] = ( ! empty( $r['count_pending_replies'] ) && current_user_can( 'edit_others_replies' ) ) ? (int) $all_replies->{$pending} : 0;

Index: src/includes/forums/functions.php
===================================================================
--- src/includes/forums/functions.php	(revision 6132)
+++ src/includes/forums/functions.php	(working copy)
@@ -2189,6 +2189,10 @@

 		/** Default ***********************************************************/

+		// Setup the post_stati array
+		$post_stati = array();
+		$post_stati[] = bbp_get_public_status_id();
+
 		// Get any existing post status
 		$post_stati = $posts_query->get( 'post_status' );

Index: src/includes/forums/template.php
===================================================================
--- src/includes/forums/template.php	(revision 6132)
+++ src/includes/forums/template.php	(working copy)
@@ -693,6 +693,7 @@
 	}

 	// Setup post status array
+	$post_stati = array();
 	$post_stati[] = bbp_get_public_status_id();

 	// Super admin get whitelisted post statuses
Index: src/includes/users/template.php
===================================================================
--- src/includes/users/template.php	(revision 6132)
+++ src/includes/users/template.php	(working copy)
@@ -1709,6 +1709,9 @@
 		// Neither a reply nor a topic, so could be a revision
 		if ( ! empty( $r['post_id'] ) ) {

+			// Setup author_links array
+			$author_links = array();
+
 			// Generate title with the display name of the author
 			if ( empty( $r['link_title'] ) ) {
 				$r['link_title'] = sprintf( ! bbp_is_reply_anonymous( $r['post_id'] ) ? __( 'View %s\'s profile', 'bbpress' ) : __( 'Visit %s\'s website', 'bbpress' ), get_the_author_meta( 'display_name', $user_id ) );
